pages/faq.md

```diff
@@ -113,7 +113,7 @@ The `getfacl` command is an easy way to see the permissions of a file or directo
 
 During installation you may see an error such as `"conda.exe: error while loading shared libraries: libz.so.1: failed to map segment from shared object: Operation not permitted"`  
 
-This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or Panasas scratch directory and specify that location in the installation command. For example:
+This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or global scratch directory and specify that location in the installation command. For example:
 ```
 mkdir /projects/academic/<group_name>/condatemp
```

Comment:
mkdir /projects/academic/{YourGroupName}/condatemp

Reply:
removed this question altogether as we no longer support Anaconda